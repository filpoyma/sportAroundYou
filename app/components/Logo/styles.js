import { css } from 'styled-components';

const LogoStyles = css``;

export default LogoStyles;
