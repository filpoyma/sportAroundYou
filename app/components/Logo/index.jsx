import React from 'react';

import logo from '@/images/logo.svg?file';
import injectStyles from '@/utils/injectStyles';

import styles from './styles';

const Logo = ({ ...rest }) => <img src={logo} alt="ROC Innovation Center | DigitalLab" {...rest} />;

Logo.propTypes = {};

Logo.defaultProps = {};

Logo.displayName = 'LogoComponent';

export default injectStyles(Logo, styles);
