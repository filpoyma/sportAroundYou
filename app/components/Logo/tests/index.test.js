import { render } from '@testing-library/react';
import React from 'react';

import Logo from '../index';

describe('<Logo />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(<Logo />);

    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild }
    } = render(<Logo />);

    expect(firstChild).toMatchSnapshot();
  });
});
