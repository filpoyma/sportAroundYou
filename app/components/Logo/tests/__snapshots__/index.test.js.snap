// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Should render and match the snapshot 1`] = `
<img
  alt="ROC Innovation Center | DigitalLab"
  class="LogoComponent-sc-1cbh0i-0 iCLpkg"
  src="IMAGE_MOCK"
/>
`;
